frontend/src/Dashboard/CrossingListPage/CrossingListHeader/CrossingListHeader.js

```diff
@@ -0,0 +1,75 @@
+import React, { Component } from 'react';
+import './CrossingListHeader.css';
+import {ContainerQuery} from 'react-container-query';
+import classnames from 'classnames';
+import FontAwesome from 'react-fontawesome';
+
+const query = {
+  'fullsize': { minWidth: 768 },
+  'smallsize': { maxWidth: 767 }
+};
+
+class CrossingListHeader extends Component {
+  render() {
+    return (
+      <ContainerQuery query={query}>
+        {(params) => (
+          
+          <div className={classnames(params, 'CrossingListHeader')}>
+            <div className={classnames(params, 'CrossingListSearch')}>
+              <input type="text" className={classnames(params, 'CrossingListSearchInput')} placeholder="Search your crossings"/>
+              <div className={classnames(params, 'CrossingListSearchButton')}>
+                <FontAwesome name="search" />
```

Comment:
I definitely agree that SVGs are a better way to go for this. I started using FontAwesome when I was working on the StatusToggle control since we didn't have any SVG assets and it seemed like a quick way to get the UI close to what we're going for.

I'll add the ariaLabels to the FontAwesome icons and we can get moving everything to SVG into the backlog if that works.